jobs/periodic/sync-to-webflow/sync-courses.ts

```diff
@@ -0,0 +1,163 @@
+import { createNewItem, deleteItems, emptyMetadata, getCollectionItems, publishItems, WebflowMetadata } from './webflow-adapter';
+import { diff, hash, mapDBIdToId, DBIdMap } from './diff';
+import { Logger } from 'log4js';
+import moment, { Moment } from 'moment';
+import { accessURLForKey } from '../../../common/file-bucket';
+import { WebflowSubcourse, getWebflowSubcourses } from './queries';
+import { lectureDTOFactory } from './sync-lectures';
+
+// This is needed so that the weekday will be translated properly.
+moment.locale('de');
+
+const collectionId = process.env.WEBFLOW_COURSE_COLLECTION_ID;
+const lectureCollectionId = process.env.WEBFLOW_LECTURE_COLLECTION_ID;
```

Comment:
What if these are not set? I would expect an error to be logged, or the job to do nothing (i.e. in the dev environments)